import { createError } from "../utils/error.js";
import User from "../models/User.js";

export const update = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const deleteUser = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const getUser = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const subscribe = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const unsubscribe = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const like = (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
export const dislike = async (req, res, next) => {
  try {
  } catch (error) {
    next(error);
  }
};
